app/_locales/en/messages.json

```diff
@@ -246,6 +246,9 @@
   "addIPFSGateway": {
     "message": "Add your preferred IPFS gateway"
   },
+  "addImportAccount": {
+    "message": "Add account or hardware wallet"
```

Comment:
This seems like a very long string to have inside of a button just want to make sure that it's been signed off by content cc @coreyjanssen

Reply:
I did sign off on the text but am now doubting it! It seemed like the best option to encompass what lies behind the CTA but perhaps Add account or wallet would suffice? Wdyt?